lib/nvme/nvme_ctrlr_cmd.c

```diff
@@ -438,3 +438,18 @@ nvme_ctrlr_cmd_fw_image_download(struct spdk_nvme_ctrlr *ctrlr,
 
 	return rc;
 }
+
+inline int32_t 
```

Comment:
These new APIs should be unnecessary if the NVMe bdev code tracks its own outstanding admin commands (see comment on blockdev_nvme.c).